cms/templates/js/self-paced-due-date-editor.underscore

```diff
@@ -0,0 +1,16 @@
+<ul class="list-fields list-input datepair date-setter">
```

Comment:
FYI I'm only doing cursory reviews of the underscore file because I imagine we might be changing the UI a bit throughout this process as we refine the feature.